### PR TITLE
Address pylint findings

### PR DESCRIPTION
**Pull request by anothertobi**
This PR resolves the open pylint warnings which currently fail the pylint action for the Renovate PRs.

### CODE REVIEW

File: concierge_cli/manager.py [hunk 1]
```diff
@@ -118,6 +118,7 @@ def merge_requests(self):
     def show(self):
         """Display all merge requests found with some status information."""
         if self.labels:
+            # pylint: disable=consider-using-f-string
             print("Open merge requests matching labels: [%s]" %
                   "][".join(self.labels))
```
Comment (bittner) [inline concierge_cli/manager.py:123]:
```suggestion
            print(f"Open merge requests matching labels: [{']['.join(self.labels)}]")
```

File: concierge_cli/manager.py [hunk 2]
```diff
@@ -129,6 +130,7 @@ def show(self):
                         'success' else '✗'
             mr_status = '✓' if merge_request.merge_status == \
                         'can_be_merged' else '✗'
+            # pylint: disable=consider-using-f-string
             mr_labels = ' [%s]' % ']['.join(merge_request.labels) \
                         if merge_request.labels else ''
```
Comment (bittner) [inline concierge_cli/manager.py:135]:
```suggestion
            mr_labels = f" [{']['.join(merge_request.labels)}]" \
                        if merge_request.labels else ''
```

File: concierge_cli/manager.py [hunk 3]
```diff
@@ -138,6 +140,7 @@ def show(self):
     def merge_all(self):
         """Merge all identified merge requests."""
         if self.labels:
+            # pylint: disable=consider-using-f-string
             print("Merging merge requests that match labels: [%s]" %
                   "][".join(self.labels))
```
Comment (bittner) [inline concierge_cli/manager.py:145]:
```suggestion
            print(f"Merging merge requests that match labels: [{']['.join(self.labels)}]")
```

File: setup.py [hunk 4]
```diff
@@ -11,7 +11,8 @@
 
 def read_file(filename):
     """Get the contents of a file"""
-    with open(join(abspath(dirname(__file__)), filename)) as file:
+    path = join(abspath(dirname(__file__)), filename)
+    with open(path, encoding='UTF-8') as file:
```
Comment (bittner) [inline setup.py:15]: Note that this code would be migrated to the more modern `pathlib` by the [changes in PR #86](https://github.com/vshn/concierge-cli/pull/86/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R13). Maybe you want to merge that PR first?

Comment (psy-q) [reply setup.py]: Good idea, done.

**Review by psy-q — APPROVED** [review]

**Review by bittner — APPROVED** [review]

**Pull request merged**